api/src/main/resources/workbench-api.yaml

```diff
@@ -350,6 +350,37 @@ paths:
           description: Bad request
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/link-ras-account":
+    post:
+      tags:
+      - profile
+      consumes:
+      - application/json
+      description: Link user with a RAS account
+      operationId: linkRasAccount
+      parameters:
+      - in: body
+        name: authcode
+        description: The authorization code from RAS.
+        schema:
+          "$ref": "#/definitions/RasLinkRequestBody"
+      responses:
+        200:
+          description: The user's account linking information.
+          schema:
+            "$ref": "#/definitions/LinkedAccountInfo"
```

Comment:
The other method returns profile, shoudl this do the same? I don't know that this other information is needed

Reply:
LinkedAccountInfo is more useful as I explained in another comment, UI may need other field that don't need in profile? i.e., do you think Profile should include login.gov, era, IAL2? Or profile will keep it simple just like today's eRA field?